.env.example

```
@@ -59,6 +59,7 @@ RPC_URL_HTTP_1=
 
 # CONFIG_ vars (override via NUXT_PUBLIC_CONFIG_* in Doppler)
 NUXT_PUBLIC_CONFIG_DOCS_URL="https://docs.euler.finance/"
+NUXT_PUBLIC_CONFIG_STARGATE_URL="http://stargate.finance/"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)